pdns/README-dnsdist.md

```diff
@@ -1425,6 +1438,7 @@ instantiate a server with additional parameters
     * `clearDynBlocks()`: clear all dynamic blocks
     * `showDynBlocks()`: show dynamic blocks in force
     * `addDynBlocks(addresses, message[, seconds])`: block the set of addresses with message `msg`, for `seconds` seconds (10 by default)
+    * `addWhitelistedDynBlocks(addresses, message[, seconds[, whitelist]])`: block the set of addresses with message `msg`, for `seconds` seconds. Addresses present in the`whitelist` `NetmaskGroup` will not be blocked even if they match
```

Comment:
> Any reason you didn't add another argument to `addDynBlocks`?

I tried but couldn't make it work, because mixing `boost::optional` with a reference to an object having a `boost::noncopyable` member wasn't going well IIRC. Passing a `boost::optional<const NetmaskGroup*>` was compiling fine but the passing a `NetmaskGroup` from Lua resulted then in:

`Error during execution of maintenance function: Unable to convert parameter from no value to N5boost8optionalIPK12NetmaskGroupEE`

I'll check again because I agree it would be so much nicer :) As for the name, `addDynBlocksWithWhitelist` was actually my first idea but it seemed to verbose, perhaps I should reconsider!